src/index.html

```diff
@@ -16,17 +16,34 @@
       href="style.css"
     />
   </head>
+
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      action="#"
+      class="search__form"
+      data-qa="big"
+    >
+      <input
+        id="big-search-input"
+        class="search__input"
+        type="text"
+        placeholder="Try “Los Angeles“"
+        data-qa="keypress"
+      />
```

Comment:
The `input` elements are missing the `name` attribute. The `name` attribute is important as it specifies the name for the input element and is used to reference form data after a form is submitted.